_posts/2017-08-15-magick-10.md

```diff
@@ -0,0 +1,164 @@
+---
+name: magick_release_10
+layout: post_discourse
+title: "Magick 1.0: 🎩 ✨🐇 Advanced Graphics and Image Processing in R "
+date: 2017-08-15
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- images
+- magick
+---
+
+
+Last week version 1.0 of the [magick](https://cran.r-project.org/web/packages/magick/index.html) package has appeared on CRAN: an ambitious effort to modernize and simplify high quality image processing in R. This R package builds upon the [Magick++ STL](https://www.imagemagick.org/Magick++/STL.html) which exposes a powerful C++ API to the famous ImageMagick library.
+
+![RStudio Screenshot](https://jeroen.github.io/images/magick.png)
+
+The best place to start learning about magick is the [vignette](https://cran.r-project.org/web/packages/magick/vignettes/intro.html) which gives a brief overview of the overwhelming amount of functionality in this package.
+
+
+## Towards Release 1.0
+
+Last year around this time rOpenSci [announced](https://ropensci.org/blog/blog/2016/08/23/z-magick-release) the first release of the `magick` package: a new powerful toolkit for image reading, writing, converting, editing, transformation, annotation, and animation in R. Since the initial release there have been a several updates with additional functionality, and many useRs have started to discover the power of this package to take visualization in R to the next level.
+
+For example [Bob Rudis](https://twitter.com/hrbrmstr/status/758304420224466944) uses magick to visualize California drought data from the U.S. Drought Monitor (click on the image to go find out more):
+
+[![drought](https://jeroen.github.io/images/drought.gif)](https://rud.is/b/2016/07/27/u-s-drought-animations-with-the-witchs-brew-purrr-broom-magick/)
+
+R-ladies [Lucy D'Agostino McGowan](http://www.lucymcgowan.com/) and [Maëlle Salmon](http://www.masalmon.eu/) demonstrate how to make a beautiful collage:
+
+[![collage](http://livefreeordichotomize.com/images/we-r-ladies.jpeg)](http://livefreeordichotomize.com/2017/07/18/the-making-of-we-r-ladies/)
+
+And [Daniel P. Hadley](https://twitter.com/danielphadley/status/884845188979359744) just likes to make confusing graphics:
+
+[![travolta](http://danielphadley.com/images/Cars_Travolta.gif)](http://danielphadley.com/ggplot-Logo/)
+
+Now, 1 year later, the 1.0 release marks an important milestone: the addition of a new native graphics device (which serves as a hybrid between a magick image object and an R plot) bridges the gap between graphics and image processing in R. 
+
+This blog post explains how the magick device allows you to seamlessly combine graphing with image processing in R. You can either use it to post-process your R graphics, or draw on imported images using the native R plotting machinery. We hope that this unified interface will make it easier to produce beautiful, reproducible images with R.
```

Comment:
I don't like to overuse the backticks because the pink coloring makes it difficult to read. Therefore I use the backticks only for actual R commands, not to name the package.